config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -275,7 +275,11 @@ pull_optional() {
         # Make tests optional until everything is updated.
         echo "true"
     else
-        echo "false"
+	if [ "$repo" == "external-resizer" ] ; then
+	    echo "true"
```

Comment:
Can you add a TODO here:
"Once https://github.com/kubernetes/kubernetes/pull/129234 merges, the 1.32 jobs for resizer should start passing and then we can make those required."

Reply:
added